test/lambda/tools-container-compose-network.test.js

```diff
@@ -27,12 +33,17 @@ useNewContainer({
 });
 
 test.serial.after.always(async (test) => {
-  const network = await networkPromise;
-  createContainer.restore();
-  await network.remove();
+  const { createContainer, network } = test;
+  if (createContainer) {
+    createContainer.restore();
+  }
+  if (network) {
+    await network.remove();
+  }
 });
 
 test('Managed containers can use a compose network', async (test) => {
+  const { createContainer } = test.context;
```

Comment:
Is this line used?

Reply:
Yes it is used a few lines afterwards. 
https://github.com/lifeomic/lambda-tools/pull/184/files#diff-3d77139ea16f92b7e1ac69340fc0ab3eL41

Reply:
Oo, oops. Missed the scoping change.